### DISCUSSION TITLE
Music player?Hu

### DISCUSSION THREAD

**Discussion by smokinjo** [advplyr/audiobookshelf#3654] [Q&A]
Hi

Can ABS be used an a music player?

Thanks

Joseph

**Comment by nichwall** [comment]
You can add music files to your server and listen to them, but ABS isn't really designed for music. You can add music to the ABS server and listen to it in the app, but the ABS app does not support playback from media not on your ABS server.

You could try something like Subsonic or another music server, or use Jellyfin/Plex with a music app which supports various server back ends.

**Comment by smokinjo** [comment]
Thanks for your feedback.  I was just curious if it could.  Other software is likely better suited for it.

Thanks
